api/src/main/java/org/pmiops/workbench/api/BigQueryService.java

```diff
@@ -85,6 +87,13 @@ public Date getDate(List<FieldValue> row, int index) {
         if (row.get(index).isNull()) {
             throw new BigQueryException(500, "FieldValue is null at position: " + index);
         }
-        return java.sql.Date.from(Instant.ofEpochMilli(Double.valueOf(row.get(index).getStringValue()).longValue() * 1000));
+        try {
+            SimpleDateFormat isoFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
+            isoFormat.setTimeZone(TimeZone.getDefault());
```

Comment:
TimeZone.getDefault() can vary by implementation and where the code is deployed. I think we want to use UTC everywhere? (That's what we're doing for cohort materialization right now.)